### ISSUE TITLE
Tests won't load on local filesystem

### ISSUE THREAD

**Issue by macintoshhelper** [dwyl/learn-elm-architecture-in-javascript#25]
`GET file://code.jquery.com/qunit/qunit-1.18.0.css net::ERR_FILE_NOT_FOUND` error is given if loading the test html page from local file system (not from a live-server).

This is because `//` is used in the script sources which will use file:// instead of http://. 

`https://` could be used instead of `//`. 

```js
<script src="https://code.jquery.com/qunit/qunit-1.18.0.js"></script>
```
instead of `//code.jquery...`

**Comment by nelsonic** [comment]
@macintoshhelper _thank you_ very much for reporting this issue! ❤️ 
Do you have time to send us a Pull Request fixing it?

**Comment by macintoshhelper** [comment]
Fixed with #26 :)

**Comment by nelsonic** [comment]
@macintoshhelper thanks again! 🎉
